extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/linkHandler/YoutubeSearchQueryHandlerFactory.java

```diff
@@ -37,73 +26,11 @@ public static YoutubeSearchQueryHandlerFactory getInstance() {
 
     @Override
     public String getUrl(final String searchString,
-                         @Nonnull final List<String> contentFilters,
-                         final String sortFilter)
-            throws ParsingException, UnsupportedOperationException {
-        try {
-            if (!contentFilters.isEmpty()) {
-                final String contentFilter = contentFilters.get(0);
-                switch (contentFilter) {
-                    case VIDEOS:
-                        return SEARCH_URL + encodeUrlUtf8(searchString)
-                                + "&sp=EgIQAfABAQ%253D%253D";
-                    case CHANNELS:
-                        return SEARCH_URL + encodeUrlUtf8(searchString)
-                                + "&sp=EgIQAvABAQ%253D%253D";
-                    case PLAYLISTS:
-                        return SEARCH_URL + encodeUrlUtf8(searchString)
-                                + "&sp=EgIQA_ABAQ%253D%253D";
-                    case MUSIC_SONGS:
-                    case MUSIC_VIDEOS:
-                    case MUSIC_ALBUMS:
-                    case MUSIC_PLAYLISTS:
-                    case MUSIC_ARTISTS:
-                        return MUSIC_SEARCH_URL + encodeUrlUtf8(searchString);
-                }
-            }
-
-            return SEARCH_URL + encodeUrlUtf8(searchString) + "&sp=8AEB";
-        } catch (final UnsupportedEncodingException e) {
-            throw new ParsingException("Could not encode query", e);
-        }
-    }
-
-    @Override
-    public String[] getAvailableContentFilter() {
-        return new String[]{
-                ALL,
-                VIDEOS,
-                CHANNELS,
-                PLAYLISTS,
-                MUSIC_SONGS,
-                MUSIC_VIDEOS,
-                MUSIC_ALBUMS,
-                MUSIC_PLAYLISTS
-                // MUSIC_ARTISTS
-        };
-    }
-
-    @Nonnull
-    public static String getSearchParameter(final String contentFilter) {
-        if (isNullOrEmpty(contentFilter)) {
-            return "8AEB";
-        }
-
-        switch (contentFilter) {
-                case VIDEOS:
-                    return "EgIQAfABAQ%3D%3D";
-                case CHANNELS:
-                    return "EgIQAvABAQ%3D%3D";
-                case PLAYLISTS:
-                    return "EgIQA_ABAQ%3D%3D";
-                case MUSIC_SONGS:
-                case MUSIC_VIDEOS:
-                case MUSIC_ALBUMS:
-                case MUSIC_PLAYLISTS:
-                case MUSIC_ARTISTS:
-                    return "";
-                default:
-                    return "8AEB";
```

Comment:
We need to update the protobuf file in order to include the "allow emergency requests" field, and then always set that field to `true`. I didn't do this while rebasing because I have no idea how the protobuf file is obtained.

Reply:
could you point me more information about the emergency requests? Where is it handled in eg. the browser youtube client.